.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
 # Default owners
-*       @hashicorp/hds-engineering
\ No newline at end of file
+*       @hashicorp/hds-engineering
+
+# co-ownership of website docs
+/website/docs/ @hashicorp/hds-engineering @hashicorp/hds-design
```

Comment:
meaning that any changes to files within this directory will require approval from members of either of these teams

Reply:
is it "either" or "both"? I believe it's both _can_ but only 1 is required to unblock merging?